app/models/email_renderer.rb

```diff
@@ -4,25 +4,49 @@ def initialize(params:)
   end
 
   def subject
-    params[:title]
+    "GOV.UK Update - #{title}"
   end
 
   def body
     <<~BODY
-      There has been a change to *#{params[:title]}* on #{format_date(params[:public_updated_at])}.
+      #{change_note}: #{description}.
 
-      > #{params[:description]}
+      #{url}
+      Updated on #{public_updated_at}
 
-      **#{params[:change_note]}**
+      Unsubscribe from #{title} - #{unsubscribe_url}
     BODY
   end
 
 private
 
   attr_reader :params
 
-  def format_date(date)
-    return unless date
-    date.strftime("%H:%M %-d %B %Y")
+  def title
+    params.fetch(:title)
+  end
+
+  def base_path
+    params.fetch(:base_path)
+  end
+
+  def change_note
+    params.fetch(:change_note)
+  end
+
+  def description
+    params.fetch(:description)
+  end
+
+  def public_updated_at
+    params.fetch(:public_updated_at).strftime("%I:%M %P, %-d %B %Y")
+  end
+
+  def url
+    "#{Plek.new.website_root}#{base_path}"
+  end
+
+  def unsubscribe_url
+    "https://www.gov.uk/email/token/unsubscribe"
```

Comment:
Should we use `Plek` here too just to be consistent? I realise this link doesn't actually work anyway at the moment.

Reply:
Good point, fixed!